ui/components/multichain/account-overview/account-overview-layout.tsx

```diff
@@ -94,6 +97,10 @@ export const AccountOverviewLayout = ({
     }
     ///: END:ONLY_INCLUDE_IF
 
+    if (id === 'smartAccountUpgrade') {
```

Comment:
Minor, could we use `SMART_ACCOUNT_UPGRADE_SLIDE.id` here?

Reply:
done